app/src/main/java/com/mobiledevelopmentfoodapp/ui/theme/Color.kt

```diff
@@ -2,7 +2,7 @@ package com.mobiledevelopmentfoodapp.ui.theme
 
 import androidx.compose.ui.graphics.Color
 
-val Purple200 = Color(0xFFBB86FC)
-val Purple500 = Color(0xFF6200EE)
-val Purple700 = Color(0xFF3700B3)
+val Green200 = Color(0xFFBB86FC)
+val Green500 = Color(0xFF6200EE)
+val Green700 = Color(0xFF3700B3)
```

Comment:
How does this change impact accessibility?  Is the contrast better?  More accessible?